java/code/src/com/redhat/rhn/frontend/action/user/UserEditActionHelper.java

```diff
@@ -59,13 +60,13 @@ public ActionErrors updateDetails(User loggedInUser, User targetUser,
 
         Boolean readOnly = (form.get("readonly") != null);
         if (!targetUser.isReadOnly()) {
-            if (readOnly && targetUser.hasRole(RoleFactory.ORG_ADMIN) &&
+            if (BooleanUtils.isTrue(readOnly) && targetUser.hasRole(RoleFactory.ORG_ADMIN) &&
```

Comment:
`readOnly` should just be `boolean` as the result of `form.get("readonly") != null` is always non-null